### PR TITLE
BETA: Register amanthakur.is-a.dev

### PR DESCRIPTION
**Pull request by aman7thakurr**
Added `amanthakur.is-a.dev` using the Discord bot.

### CODE REVIEW

File: domains/amanthakur.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "aman7thakurr",
+       "email": "amant8425@gmail.com",
+       "discord": "523813766295715851"
+    },
+
+    "record": {
+        "CNAME": "aman7thakurr"
```
Comment (WilliamDavidHarrison) [inline domains/amanthakur.json:9]: Please use a valid record.

File: domains/aman.json
Comment (WilliamDavidHarrison) [inline domains/aman.json:1]: Please add records to this domain.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/amanthakur.json
Comment (aman7thakurr) [inline domains/amanthakur.json:1]: CNAME Updated

**Comment by aman7thakurr** [comment]
please review record added

File: domains/aman_thakur.json
Comment (WilliamDavidHarrison) [inline domains/aman_thakur.json:1]: We do not allow underscores in subdomain names.

**Review by wdhdev — APPROVED** [review]

**Pull request merged**